src/lib/stitching/convection/stitching.ts

```diff
@@ -32,5 +43,40 @@ export const consignmentStitchingEnvironment = (
         },
       },
     },
+
+    ConsignmentOffer: {
+      lowEstimateAmount: {
```

Comment:
There is a newer [`Money`](https://github.com/damassi/metaphysics/blob/2e1c89f763d9f15d16c02f5ca193d5d1d641048c/src/schema/v2/fields/money.ts#L112) type which includes recommended naming conventions that we can also consider. Some discussion in https://github.com/artsy/metaphysics/pull/2015.

Reply:
For this field we just need something simple / "pretty" for the UI and the `amount` helper fits the bill perfectly. It's used as a util all over.